conferences/ticketing_storage.go

```diff
@@ -0,0 +1,605 @@
+	results := Attendee{}
+	q := `SELECT id, email, coc_accepted FROM attendee`
+	args := []interface{}{}
+	if email != "" {
+		q += ` WHERE email = $1`
+		args = append(args, email)
+	}
+	if email != "" && id != 0 {
+		q += `AND WHERE id = $2`
+		args = append(args, id)
+	}
+	if email == "" && id != 0 {
+		q += ` WHERE id = $1`
+		args = append(args, id)
+	}
+	row := queryRow(ctx, tx, q, args...)
+	err := row.Scan(&results.ID, &results.Email, &results.CoCAccepted)
+	if err == sql.ErrNoRows {
+		return nil, nil
+	}
+	if err != nil {
+		return nil, fmt.Errorf("reading attendee by email: %w", err)
+	}
+
+	claims := []SlotClaim{}
+
+	rows, err := query(ctx, tx,
+		`SELECT id, ticket_id, redeemed FROM slot_claim 
+	JOIN attendees_to_slot_claims ON slot_claim.id = attendees_to_slot_claims.slot_claim_id
+	WHERE attendees_to_slot_claims.id = $1`,
+		results.ID)
+	if err != nil && err != sql.ErrNoRows {
+		return nil, fmt.Errorf("querying claims for attendee: %w", err)
+	}
+	for rows.Next() {
+		claim := SlotClaim{}
+		err := rows.Scan(&claim.ID, &claim.TicketID, &claim.Redeemed)
+		if err != nil {
+			return nil, fmt.Errorf("scanning slot_claims for attendee: %w", err)
+		}
+		claims = append(claims, claim)
+	}
+
+	results.Claims = claims
+	return &results, nil
+}
+
+// createConferenceSlot saves a slot in the database.
+func createConferenceSlot(ctx context.Context, tx *sqldb.Tx, cslot *ConferenceSlot, conferenceID int64) (*ConferenceSlot, error) {
+	var sqlSentence = `INSERT INTO conference_slot (conference_id, name, descripcion, cost, capacity, start_date, end_date, purchaseable_form, purchaseable_until, available_to_public)
+	VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) 
+	RETURNING conference_id, name, descripcion, cost, capacity, start_date, end_date, purchaseable_form, purchaseable_until, available_to_public`
+	var args = []interface{}{
+		conferenceID,
+		cslot.Name,
+		cslot.Description,
+		cslot.Cost,
+		cslot.Capacity,
+		cslot.StartDate,
+		cslot.EndDate,
+		cslot.PurchaseableFrom,
+		cslot.PurchaseableUntil,
+		cslot.AvailableToPublic,
+	}
+	/*if e.DependsOn != nil {
+		sqlSentence = `INSERT INTO conference_slot (conference_id, name, descripcion, cost, capacity, start_date, end_date, purchaseable_form, purchaseable_until, available_to_public, depends_on_id)
+		VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)
+		RETURNING id, name, descripcion, cost, capacity, start_date, end_date, purchaseable_form, purchaseable_until, available_to_public`
+		args = append(args, e.DependsOn.ID)
+	}*/
+	row := queryRow(ctx, tx, sqlSentence, args...)
+	results := ConferenceSlot{}
+	err := row.Scan(&results.ID,
+		&results.Name,
+		&results.Description,
+		&results.Cost,
+		&results.Capacity,
+		&results.StartDate,
+		&results.EndDate,
+		&results.PurchaseableFrom,
+		&results.PurchaseableUntil,
+		&results.AvailableToPublic)
+	if err != nil {
+		return nil, fmt.Errorf("creating new attendee: %w", err)
```

Comment:
Error is not relevant to function here. 
```suggestion
		return nil, fmt.Errorf("creating new conference slot: %w", err)
```